src/containerapp/azext_containerapp/custom.py

```diff
@@ -1169,7 +1169,7 @@ def _get_patchable_check_result(inspect_result, oryx_run_images):
         return result
 
     # Define the MCR repositories that are supported for patching
-    mcr_repos = ["oryx/dotnetcore", "oryx/node", "oryx/python"]
+    mcr_repos = ["oryx/dotnetcore", "oryx/node", "oryx/python", "azure-buildpacks/java"]
 
```

Comment:
could you add test for java scenario?

Reply:
I will add test for java in another PR after backend change is released. This feature is target for Ignite and I have verified it private deployment environment, we have to review and merge it ASAP.